pkg/restapi/client.go

```diff
@@ -17,8 +17,8 @@ type TaskOutput struct {
 	Header     map[string][]string    `json:"header"`
 }
 
-func newClient(config *structpb.Struct, logger *zap.Logger) (*httpclient.Client, error) {
-	c := httpclient.New("REST API", getBaseURL(config),
+func newClient(baseUrl string, config *structpb.Struct, logger *zap.Logger) (*httpclient.Client, error) {
+	c := httpclient.New("REST API", baseUrl,
 		httpclient.WithLogger(logger),
 	)
```

Comment:
We can also remove the `baseURL` parameter. `resty`'s `Execute` method can be called with an absolute URL if the `BaseURL` method is unset. I don't know if passing an empty string would be enough or if we'd need to modify any of the fields or call any of the functions in `c` but it'd be easy to find out
```suggestion
func newClient(config *structpb.Struct, logger *zap.Logger) (*httpclient.Client, error) {
	c := httpclient.New("REST API", "",
		httpclient.WithLogger(logger),
	)
```

Reply:
Oh, yeah, empty string can work, let me update it

Reply:
updated